backend/substrapp/tasks/k8s_backend.py

```diff
@@ -263,19 +263,24 @@ def k8s_build_image(path, tag, rm):
         if REGISTRY_SCHEME == 'http':
             args.append('--insecure')
 
-        pod_security_context = None
-        container_security_context = None
+        # https://github.com/GoogleContainerTools/kaniko/issues/778
+        capabilities = ['CHOWN', 'SETUID', 'SETGID', 'FOWNER', 'DAC_OVERRIDE']
+        pod_security_context = get_pod_security_context(root=True)
+        container_security_context = get_security_context(root=True, add_capabilities=capabilities)
 
     elif IMAGE_BUILDER == 'makisu':
+        # makisu build can be launched without privilege but
+        # it needs to be root
         image = 'gcr.io/uber-container-tools/makisu:v0.2.0'
         command = None
         mount_path_dockerfile = '/makisu-context'
-        mount_path_cache = None
+        mount_path_cache = '/makisu-storage'
         args = [
             'build',
             f'--push={REGISTRY}',
             f'-t={tag}:substra',
             '--modifyfs=true',
+            f'--storage="{mount_path_cache}"'
```

Comment:
Nit: I don't think you need this line

```
 --storage string                  Directory that makisu uses for temp files and cached layers. Mount this path for better caching performance. If modifyfs is set, default to /makisu-storage; Otherwise default to /tmp/makisu-storage
```

Reply:
I prefer to do it explicitly as if we change modifyfs, makisu storage will be in /tmp/makisu-storage